src/cmakeProject.ts

```diff
@@ -276,6 +276,7 @@ export class CMakeProject {
             buildPreset,
             lightNormalizePath(this.folder.uri.fsPath || '.'),
             this.sourceDir,
+            this.workspaceContext.config,
```

Comment:
Can you just pass the `config.parallelJobs` instead of the whole `config`?

Reply:
This was my rationale:

> I decided to pass the config class to the function which generates the default build preset, such that future changes according the defaultBuildPreset with regard to configuration values can be done easily without further cluttering the function signature.

But of course it can be changed.